tests/scripts/mbedtls_test.py

```diff
@@ -310,7 +310,10 @@ def run_test(self, name, function_id, dependencies, args):
 
         param_bytes, length = self.test_vector_to_bytes(function_id,
                                                         dependencies, args)
-        self.send_kv(''.join('{:02x}'.format(x) for x in length), ''.join('{:02x}'.format(x) for x in param_bytes))
+        self.send_kv(
+            ''.join('{:02x}'.format(x) for x in length),
+            ''.join('{:02x}'.format(x) for x in param_bytes)
+        )
```

Comment:
How about
```
params = length + param_bytes
self.send_kv('.join('{:02x}'.format(x) for x in params))
```

Reply:
Wouldn't that send one string, rather than two separate strings?

Reply:
Oh right. Never mind.